server/controllers/favorites.js

```diff
@@ -0,0 +1,35 @@
+const { default: mongoose } = require('mongoose');
+const Favorite = require('../models/Favorite.js')
+
+const getFavorites = async (req, res) => {
+  try {
+    const Favorites = await Favorite.find();
+    res.status(200).json(Favorites);
+  } catch (error) {
+    res.status(404).json({message: error.message})
```

Comment:
Technically a 404 status is probably not the best one to send in most cases --- usually 404 is most appropriate when a findById/findByPk is triggered and no matching resource can be found. I don't think keeping these as 404 is the end of the world for now, but later on you may want to do some reading into status codes and decide which status code is most appropriate in a given situation.